src/um/msdelta.rs

```diff
@@ -0,0 +1,283 @@
+// Copyright © 2018 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+use ctypes::__int64;
+use shared::basetsd::SIZE_T;
+use shared::minwindef::{BOOL, DWORD, FILETIME, LPCVOID, LPVOID, UCHAR};
+use um::wincrypt::ALG_ID;
+use um::winnt::{LPCSTR, LPCWSTR};
+pub const DELTA_FILE_SIZE_LIMIT: SIZE_T = 32 * 1024 * 1024;
+pub const DELTA_OPTIONS_SIZE_LIMIT: SIZE_T = 128 * 1024 * 1024;
+UNION!{union DELTA_INPUT_U {
+    [usize; 1],
+    lpcStart lpcStart_mut: LPCVOID,
+    lpStart lpStart_mut: LPVOID,
+}}
+STRUCT!{struct DELTA_INPUT {
+    u: DELTA_INPUT_U,
+    uSize: SIZE_T,
+    Editable: bool,
+}}
+pub type LPDELTA_INPUT = *mut DELTA_INPUT;
+pub type LPCDELTA_INPUT = *const DELTA_INPUT;
+STRUCT!{struct DELTA_OUTPUT {
+    lpStart: LPVOID,
+    uSize: SIZE_T,
+}}
+pub type LPDELTA_OUTPUT = *mut DELTA_OUTPUT;
+pub type LPCDELTA_OUTPUT = *const DELTA_OUTPUT;
+pub type DELTA_FILE_TYPE = __int64;
+pub const DELTA_FILE_TYPE_RAW: DELTA_FILE_TYPE = 0x00000001;
+pub const DELTA_FILE_TYPE_I386: DELTA_FILE_TYPE = 0x00000002;
+pub const DELTA_FILE_TYPE_IA64: DELTA_FILE_TYPE = 0x00000004;
+pub const DELTA_FILE_TYPE_AMD64: DELTA_FILE_TYPE = 0x00000008;
+pub const DELTA_FILE_TYPE_CLI4_I386: DELTA_FILE_TYPE = 0x00000010;
+pub const DELTA_FILE_TYPE_CLI4_AMD64: DELTA_FILE_TYPE = 0x00000020;
+pub const DELTA_FILE_TYPE_CLI4_ARM: DELTA_FILE_TYPE = 0x00000040;
+pub const DELTA_FILE_TYPE_CLI4_ARM64: DELTA_FILE_TYPE = 0x00000080;
+pub const DELTA_FILE_TYPE_SET_RAW_ONLY: DELTA_FILE_TYPE = DELTA_FILE_TYPE_RAW;
+pub const DELTA_FILE_TYPE_SET_EXECUTABLES_1: DELTA_FILE_TYPE = DELTA_FILE_TYPE_RAW
+                                                             | DELTA_FILE_TYPE_I386
+                                                             | DELTA_FILE_TYPE_IA64
+                                                             | DELTA_FILE_TYPE_AMD64;
```

Comment:
Do not use aligned indentation.

Reply:
https://github.com/retep998/winapi-rs/pull/710/commits/8ac2ea3eecbffcb3021a8b2a7351ddacb3d73040 Fixed it.